crates/superposition/src/organisation/handlers.rs

```diff
@@ -0,0 +1,148 @@
+use super::types::{
+    CreateOrganisationRequest, CreateOrganisationResponse, OrganisationResponse,
+};
+use actix_web::{
+    get, post,
+    web::{self, Json, Query},
+    HttpResponse, Scope,
+};
+use chrono::Utc;
+use diesel::prelude::*;
+use idgenerator::IdInstance;
+use service_utils::service::types::DbConnection;
+use superposition_types::{
+    custom_query::PaginationParams, result as superposition, PaginatedResponse,
+};
+
+use superposition_types::database::{
+    models::organisation::Organisation, schema::organisation::dsl::organisation,
+};
+
+pub fn endpoints() -> Scope {
+    Scope::new("")
+        .service(create_organisation)
+        .service(list_organisations)
+        .service(get_organisation)
+}
+
+#[post("")]
+pub async fn create_organisation(
+    req: web::Json<CreateOrganisationRequest>,
+    db_conn: DbConnection,
+) -> superposition::Result<HttpResponse> {
+    let DbConnection(mut conn) = db_conn;
+    
+    // Generating a numeric ID from IdInstance and prefixing it with `orgid`
+    let numeric_id = IdInstance::next_id();
```

Comment:
we planned to keep the org id of fixed length 25 right ?
to allow for some buffer in future?

`orgid` is 5 characters, wanted to just know that how is `IdInstance::next_id()` making sure that the length of the string given is of 20 characters exact ?

or are we keeping the id value dynamic and not fixed length, because I was of the impression that we planned to keep the id fixed length